recipes/rdflib-pyld-compat/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "rdflib-pyld-compat" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/rdflib-pyld-compat-{{ version }}.tar.gz
+  sha256: f230d565d221efd9b28497034571c9c15e03a6fa9409b77277ca9295b94d072d
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: True  # [osx]
```

Comment:
What's the reason to skip `osx`? Seems like this could be a `noarch: python` recipe.

Reply:
:lau

Reply:
Yep, I've reintroduced it in the last commit. Thanks Uwe!
That was just an artifact of me poking around and I think I've found an error with the [conda-build](https://github.com/conda/conda-build/issues/5566)<>OSx M3?